test/http-invocation.test.js

```diff
@@ -193,6 +193,182 @@ describe('HttpInvocation', function() {
       inv.transformResponse(res, body, cb);
     }
   });
+
+  describe('createRequest', function() {
+    it('should create a simple request', function() {
+      var method = givenSharedStaticMethod();
+      method.getEndpoints = function() {
+        return [createEndpoint()];
+      };
+      var inv = givenInvocation(method, { ctorArgs: [], args: [], baseUrl: 'http://base' });
+
```

Comment:
line 199:203 could be extracted to a single function as I see them being repeated.